### ISSUE TITLE
FlareSolverr not by-passing CloudFlare

### ISSUE THREAD

**Issue by S0ly** [FlareSolverr/FlareSolverr#615]
BUG / ERROR

* **FlareSolverr version**: 2.2.10
* **Last working FlareSolverr version**: i'm new so 2.2.10
* **Operating system**: ubuntu / ubuntu server
* **Are you using Docker**: yes
* **FlareSolverr User-Agent (see log traces or / endpoint)**: idk see logs
* **Are you using a proxy or VPN?** no
* **Are you using Captcha Solver:** no]
* **If using captcha solver, which one:**
* **URL to test this issue:** https://cpasbiens3.fr/

### Description
i just want to add flaresolverr to prowlarr so it can go on the website above but it does not by-pass cloudflare like entended

### Logged Error Messages
prawlarr error message wen i want to add the website to the indexers : Unable to access cpasbiens3.fr, blocked by CloudFlare Protection.

[prowlarr (1).txt](https://github.com/FlareSolverr/FlareSolverr/files/10202783/prowlarr.1.txt)
[flaresolverlogs.txt](https://github.com/FlareSolverr/FlareSolverr/files/10202790/flaresolverlogs.txt)

### Screenshots
![image](https://user-images.githubusercontent.com/86328249/206918913-0c7b87ca-4456-4a93-9eaf-15c6271b1144.png)
<img width="536" alt="image" src="https://user-images.githubusercontent.com/86328249/206918923-767cf91c-4ce0-45ca-861a-6e113f7e9b4b.png">

Everything is setuped correctly i have even reinstalled some things please help

**Comment by ilike2burnthing** [comment]
Try the v3 beta https://github.com/FlareSolverr/FlareSolverr/issues/522

**Comment by S0ly** [comment]
ok thanks i will try

**Comment by S0ly** [comment]
i have tried the V3 beta on prowlarr nothing worked it just say : Failed to test proxy: 172.17.0.3 (docker ip)

here is the logs of the beta :
```
2022-12-14 00:18:46 INFO     FlareSolverr 3.0.0.beta3
2022-12-14 00:18:46 INFO     Testing web browser installation...
2022-12-14 00:19:56 INFO     FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux aarch64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
2022-12-14 00:19:56 INFO     Test successful
2022-12-14 00:19:59 INFO     Serving on http://0.0.0.0:8191
2022-12-14 00:20:14 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:20:14 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:20:14 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:20:14 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:22:01 INFO     Challenge not detected!
2022-12-14 00:22:01 INFO     Challenge not detected!
2022-12-14 00:22:06 INFO     Response in 111.882 s
2022-12-14 00:22:06 INFO     172.17.0.1 POST http://localhost:8191/v1 200 OK
2022-12-14 00:22:07 INFO     Response in 113.104 s
2022-12-14 00:22:07 INFO     172.17.0.1 POST http://localhost:8191/v1 200 OK
2022-12-14 00:22:29 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:22:29 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:22:29 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:22:29 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:23:43 INFO     Incoming request => POST /v1 body: {'maxTimeout': 180000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:23:43 INFO     Incoming request => POST /v1 body: {'maxTimeout': 180000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:23:45 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:23:45 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:24:07 INFO     Challenge not detected!
2022-12-14 00:24:08 INFO     Challenge not detected!
2022-12-14 00:24:12 INFO     Response in 103.295 s
2022-12-14 00:24:12 INFO     172.17.0.1 POST http://172.17.0.3:8191/v1 200 OK
2022-12-14 00:24:13 INFO     Response in 104.26 s
2022-12-14 00:24:13 INFO     172.17.0.1 POST http://172.17.0.3:8191/v1 200 OK
2022-12-14 00:25:43 INFO     Challenge not detected!
2022-12-14 00:25:44 INFO     Challenge not detected!
2022-12-14 00:25:47 INFO     Response in 124.496 s
2022-12-14 00:25:48 INFO     172.17.0.1 POST http://172.17.0.3:8191/v1 200 OK
2022-12-14 00:25:48 INFO     Response in 124.916 s
2022-12-14 00:25:48 INFO     172.17.0.1 POST http://172.17.0.3:8191/v1 200 OK
root@raspberrypi:/home/pi# docker logs flaresolverrbeta3
2022-12-14 00:18:46 INFO     FlareSolverr 3.0.0.beta3
2022-12-14 00:18:46 INFO     Testing web browser installation...
2022-12-14 00:19:56 INFO     FlareSolverr User-Agent: Mozilla/5.0 (X11; Linux aarch64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/106.0.0.0 Safari/537.36
2022-12-14 00:19:56 INFO     Test successful
2022-12-14 00:19:59 INFO     Serving on http://0.0.0.0:8191
2022-12-14 00:20:14 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:20:14 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:20:14 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:20:14 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:22:01 INFO     Challenge not detected!
2022-12-14 00:22:01 INFO     Challenge not detected!
2022-12-14 00:22:06 INFO     Response in 111.882 s
2022-12-14 00:22:06 INFO     172.17.0.1 POST http://localhost:8191/v1 200 OK
2022-12-14 00:22:07 INFO     Response in 113.104 s
2022-12-14 00:22:07 INFO     172.17.0.1 POST http://localhost:8191/v1 200 OK
2022-12-14 00:22:29 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:22:29 INFO     Incoming request => POST /v1 body: {'maxTimeout': 60000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:22:29 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:22:29 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:23:43 INFO     Incoming request => POST /v1 body: {'maxTimeout': 180000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:23:43 INFO     Incoming request => POST /v1 body: {'maxTimeout': 180000, 'cmd': 'request.get', 'url': 'https://prowlarr.servarr.com/v1/ping', 'userAgent': 'Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/81.0.4044.138 Safari/537.36', 'proxy': {}}
2022-12-14 00:23:45 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:23:45 WARNING  Request parameter 'userAgent' was removed in FlareSolverr v2.
2022-12-14 00:24:07 INFO     Challenge not detected!
2022-12-14 00:24:08 INFO     Challenge not detected!
2022-12-14 00:24:12 INFO     Response in 103.295 s
2022-12-14 00:24:12 INFO     172.17.0.1 POST http://172.17.0.3:8191/v1 200 OK
2022-12-14 00:24:13 INFO     Response in 104.26 s
2022-12-14 00:24:13 INFO     172.17.0.1 POST http://172.17.0.3:8191/v1 200 OK
2022-12-14 00:25:43 INFO     Challenge not detected!
2022-12-14 00:25:44 INFO     Challenge not detected!
2022-12-14 00:25:47 INFO     Response in 124.496 s
2022-12-14 00:25:48 INFO     172.17.0.1 POST http://172.17.0.3:8191/v1 200 OK
2022-12-14 00:25:48 INFO     Response in 124.916 s
2022-12-14 00:25:48 INFO     172.17.0.1 POST http://172.17.0.3:8191/v1 200 OK
```

**Comment by S0ly** [comment]
the flaresolverr normal version is not really working too because its detected by cloudflare

**Comment by ilike2burnthing** [comment]
The FlareSolverr logs seems to show it's working fine, not sure if the issue like with Prowlarr or just your configuration.

**Comment by S0ly** [comment]
did it work for you whit prowlarr ?

**Comment by ngosang** [comment]
Released FlareSolverr 3.0.0 and Jackett 0.20.2486 => https://github.com/FlareSolverr/FlareSolverr/issues/650
